deps/config/types.go

```diff
@@ -548,3 +563,37 @@ type ApisConfig struct {
 	// cat ~/.lotusminer/keystore/MF2XI2BNNJ3XILLQOJUXMYLUMU | jq -r .PrivateKey
 	StorageRPCSecret string
 }
+
+// HTTPConfig represents the configuration for an HTTP server.
+type HTTPConfig struct {
+	// DomainName specifies the domain name that the server uses to serve HTTP requests.
+	DomainName string
+
+	// ListenAddr is the address that the server listens for HTTP requests.
+	ListenAddr string
+
+	// ReadTimeout is the maximum duration for reading the entire or next request, including body, from the client.
+	ReadTimeout time.Duration
+
+	// WriteTimeout is the maximum duration before timing out writes of the response to the client.
+	WriteTimeout time.Duration
```

Comment:
Are we later able to set this higher for responses from e.g. the /piece endpoint?

Reply:
Yes. I will try to find better write defaults once this is merged. More practical ones.